.github/workflows/ci.yml

```diff
@@ -32,8 +32,8 @@ jobs:
           path: ~/.sbt
           key: sbt-${{ hashFiles('**/build.sbt') }}
       - name: Check formatting
-        if: startsWith(matrix.scala, '2.12.') && (matrix.java == '8')
-        run: sbt ++$SCALA_VERSION scalafmtCheckAll
+        if: startsWith(matrix.scala, '2.12.') && (matrix.java == 'adopt@1.8')
+        run: sbt ++$SCALA_VERSION test:scalafmt::test
```

Comment:
This line needs to be reverted when we merge to series/0.21.  We would have a hell of a merge conflict if we upgraded scalafmt on 0.20.